source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,273 @@
+---
+title: Direct Debit
+weight: 75
+---
+
+# GOV.UK Pay Variable Direct Debit
+
+Before reading this section, you should be familiar with the [__Making
+payments__ section](/payment_flow_overview/#payment-flow). You should also
+have [integrated your service with the GOV.UK Pay
+API](/integrate_with_govuk_pay/#how-to-integrate-with-the-gov-uk-pay-api).
+
+To use GOV.UK Pay for variable Direct Debit transactions, you should:
+
+* request a GOV.UK Pay Direct Debit test account [from our
+support team](/support_contact_and_more_information/#contact-us).
+
+* add support for the Direct Debit endpoints to your existing GOV.UK Pay
+integration
+
+* sign in to your GOV.UK Pay Direct Debit account and follow the instructions
+to set up an account with our Direct Debit supplier
+
+## GOV.UK Pay ‘agreements’
+
+In order to support Direct Debit, GOV.UK Pay now uses ‘agreements’. An
+agreement represents an individual payer’s consent for future payments using a
+particular funding source.
+
+For Direct Debit transactions, GOV.UK Pay fulfils an agreement by registering
+a Direct Debit mandate with the payer’s bank.
+
+## Synchronous and asynchronous payment methods
+
+Card payments are a synchronous payment method. When a user
+completes a card payment using GOV.UK Pay, the platform immediately confirms
+to your service whether the payment has succeeded or failed. If a user
+pays by card, you can immediately release your service to the user .
+
+Direct Debit payments are an asynchronous payment method. When a user
+completes a Direct Debit payment process using GOV.UK Pay, the platform makes
+a request to their bank to set up the Direct Debit mandate and transfer the
+funds. It takes 3-4 days to confirm the success of the payment.
+
+## One-off payments using Direct Debit
+
+To set up a Direct Debit mandate to take a single payment:
+
+* create an API key linked to your Direct Debit service
+
+* follow the instructions in the [‘Payment Flow’ section](/payment_flow)
+
+You cannot take further payments from users using this method without creating
+a new payment. This requires users to re-enter their payment details.
+
+## Variable payments using Direct Debit
+
+A variable agreement allows you to collect payments of any amount at any time.
+For example, you may want to collect:
+
+* money each time a service is provided
+
+* payments where the amounts vary according to service usage
+
+* regular payments where you do not know in advance how much each payment will be
+
+You could also use the variable functionality to simulate a recurring or
+scheduled Direct Debit payment, by sending an API call on a set schedule.
+Users would be emailed each time a payment is collected against their mandate.
+
+You are responsible for managing variable payments using the GOV.UK Pay API.
```

Comment:
i struggled a bit to understand why this final line was in there - i think it's to say that we won't take a future schedule and execute it, but i had to scratch my head a little bit to think why it's in there

Reply:
Yeah I thought the same; I've deleted it.

Reply:
Can always add something back in after.